<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphEdgeMenu</class>
 <widget class="QDialog" name="GraphEdgeMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Властивості ребра</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/app.ico</normaloff>:/img/app.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
font: 12pt &quot;Segoe UI&quot;;
color: #f5f5f5;
}

QWidget {
background: #333;
}

QPushButton, QLineEdit, QSpinBox {
padding: 8px 16px;
background: #424242;
border: none;
border-radius: 3px;
color: #f5f5f5;
font-size: 14px;
min-height: 21px;
}

QPushButton:hover {
background: #212121;
}

QPushButton:disabled, QLineEdit:disabled {
background: #2d2d2d;
color: #707070;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Вага ребра</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="edgeWeight">
     <property name="styleSheet">
      <string notr="true">QSpinBox::up-button,
QSpinBox::down-button {
	border: none;
	display: none;
	height: 0;
	width: 0;
}</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	color: #EF5350;
	background: #333;
    font: 12pt &quot;Segoe UI&quot;;
	padding-left: 2px;
}</string>
       </property>
       <property name="text">
        <string>Видалити ребро</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/trash.svg</normaloff>:/img/trash.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>92</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton {
font: 12pt &quot;Segoe UI&quot;;
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Зберегти</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Відміна</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
